### ISSUE TITLE
reproduction issues

### ISSUE THREAD

**Issue by seominseok0429** [lxtGH/PFSegNets#9]
When the iSAID val set is verified using the iSAID 69.9 pth uploaded to Google, the performance of 66.3 comes out.

Did you do the same with the training data when creating the test data? Or is there something different?

Can you check the pth file you uploaded again?

**Comment by seominseok0429** [comment]
![Screen Shot 2021-12-31 at 10 53 42 AM](https://user-images.githubusercontent.com/33244972/147798606-c66bb269-6fc1-4918-872c-757b4706284a.png)

**Comment by hehao13** [comment]
First, thanks for your attention. 

We tested the uploaded checkpoint with this codebase. It turns out that the mIoU on the iSAID validation set is 66.97.
<img width="672" alt="image" src="https://user-images.githubusercontent.com/46488267/147809725-c99a89be-c410-4e5d-8674-aa4bb261c04a.png">

Maybe there are some problems regarding datasets, annotations.
